lib/index.ts

```diff
@@ -42,10 +48,9 @@ function parse(opt, callback) {
       callback(e, null);
     });
 
-  } else {
+  } else if (opt.xmlContent) {
     // parse the xml provided by the api client.
-    _parseWithXml2js(opt.xmlContent).then(function(result) {
-      delete result.xmlContent;
```

Comment:
We should preserve this line for backward compatibility.

Reply:
Makes sense. Made this change, and had to change the type of `ParsedOutput` to incorporate this change, otherwise typescript won't let me delete a property which is not present on the object according to the types.

Reply:
Looks good. I believe in next major version bump we might want to standardize how `xml` is stored when read from `pom` or when provided directly.